### DISCUSSION TITLE
GitHub Support haven't Answered me for over 3 Weeks

### DISCUSSION THREAD

**Discussion by tortuch611** [community/community#118778] [New to GitHub]
My computer has been hacked 4 weeks ago, and the hacker changed email of my LinkedIn and create so many spam repositories in my github account. Already linkedin account has been restored by support team, but github account is flagged yet even though I remove all repositories. It has been 3 weeks now and still no response, which is actually crazy, I need help as soon as possible because, at the moment, no one can see my profile, repositories, anything, no GitHub pages, I can't even sign into Git or GitHub on my computer anymore in different software, like vscode. I can't stress enough how bad this is for me because of the organization I'm helping can't see what I'm doing nor could I even push code to their repositories, but I figured out something that I can, but I am also trying to find a job and no employer can see any of my hard work over the couple years, this is horrible. I am getting really pissed now, this is extremely frustrating and I don't even know if they were able to see my last support ticket. I contacted GitHub even emailed a member of GitHub and still no response, this is actually ridiculous. I seriously don't know if they're ignoring me on purpose because I am really trying to reach them or maybe GitHub did this on purpose, but nahhh, can't be. I usually created other accounts using Github Account. So now I can't log in to those accounts and I can't do anything anymore. This all just caused unnecessary frustration and wasted time... I am losing respect for GitHub and I really don't want to, I really need my account back.

I am on a different account. I am just trying to reach them in whatever way possible and hoping a GitHub member sees this, since I can't anything on my main account. I do not want to start using this one because all the stars on my Repos, organizations and just being able to see my history, etc. I don't know... this is all very important to me.

My Real Account:
https://github.com/tortuc

![image](https://github.com/community/community/assets/166799092/5d9cb6da-2add-4920-9850-0edadb1e7302)

**Comment by asmawi117** [comment]
same issue brother, i've been waiting for 2 months still hasn't got any response from support team

**Reply by tortuch611** [reply]
They didn't answer yet, this is really disappointed.
